modules/__tests__/describePopState.js

```diff
@@ -18,6 +18,14 @@ const describePopState = (createHistory) => {
 
       window.history.back()
     })
+
+    it('is called when only a hash is navigated to', (done) => {
+      unlisten = history.listenBefore(() => {
+        done()
```

Comment:
Use `listen` instead of `listenBefore`. Also, assert that the listener is called exactly once.